### PR TITLE
Done HW10

### PR DESCRIPTION
**Pull request by DmitriyBabii**

### CODE REVIEW

File: src/main/java/com/alevel/hw10/models/Passenger.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package com.alevel.hw10.models;
+
+import java.util.Random;
+
+public class Passenger {
+    private final static Random rand = new Random();
+    private final static String[] names = {
```
Comment (Fabolo12) [inline src/main/java/com/alevel/hw10/models/Passenger.java]: Constant names in capital letters

File: src/main/java/com/alevel/hw10/Main.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+        passengerCar.addPassenger();
+        passengerCar.addPassenger();
+        passengerCar.addPassenger();
+        passengerCar.addPassenger();
+
+        System.out.println(passengerCar.getTraffic());
+        System.out.println(truck.getTraffic());
+
+        Recovery car = new PassengerCar(Series.AX, 242, Color.RED);
```
Comment (Fabolo12) [inline src/main/java/com/alevel/hw10/Main.java:19]: Good, but I want to see the implementation: `Recovery recovery = new Recovery()`

**Review by Fabolo12 — APPROVED** [review]

File: src/main/java/com/alevel/hw10/Main.java (hunk 2 above)
Comment (DmitriyBabii) [reply src/main/java/com/alevel/hw10/Main.java:19]: Added second option

**Review by Fabolo12 — APPROVED** [review]
10

**Pull request merged**